packages/code/src/renderingPlugin.tsx

```diff
@@ -136,8 +153,10 @@ export function LineTableRow({
           className={cx(
             cellStyle,
             css`
-              padding-right: 24px;
               user-select: none;
+              text-align: right;
+              padding-left: ${spacing[2]}px;
+              padding-right: 0;
               color: ${highlighted ? highlightedNumberColor : numberColor};
```

Comment:
In axe devtools, I'm seeing that this color does not have sufficient contrast

Reply:
Yeah, I mostly left this unchanged here, though IIRC I slightly bumped the contrast in either light or dark mode. The line numbers are decorative, so I'm not sure what the best approach is here to make the devtools pass. It's something we can address down the line, given it's not a regression or goal of this PR.

Reply:
Fair, we should document that this is an outstanding issue somewhere though

Reply:
https://jira.mongodb.org/browse/PD-1011